sparse/unit_test/Test_Sparse_block_gauss_seidel.hpp

```diff
@@ -97,29 +97,29 @@ int run_block_gauss_seidel_1(
   const int apply_count   = 100;
 
   if (!skip_symbolic) {
-    KSExp::block_gauss_seidel_symbolic(&kh, num_rows_1, num_cols_1, block_size, input_mat.graph.row_map,
-                                       input_mat.graph.entries, is_symmetric_graph);
+    KokkosSparse::block_gauss_seidel_symbolic(&kh, num_rows_1, num_cols_1, block_size, input_mat.graph.row_map,
```

Comment:
Could now get rid of KSExp namespace alias since it's not used anywhere else.

Reply:
Oh right that makes sense, will go clean that up as well